src/CommunityToolkit.Maui/Converters/IsInRangeConverter.shared.cs

```diff
@@ -1,71 +1,34 @@
-﻿using System.Globalization;
+﻿// Ignore Spelling: Bindable
+using System.Globalization;
 
 namespace CommunityToolkit.Maui.Converters;
 
 /// <summary>Converts the incoming value to a <see cref="bool"/> indicating whether or not the value is within a range.</summary>
-public sealed class IsInRangeConverter : IsInRangeConverter<object>
-{
-}
+public sealed class IsInRangeConverter : IsInRangeConverter<IComparable, object>;
 
 /// <summary>Converts the incoming value to a <see cref="bool"/> indicating whether or not the value is within a range.</summary>
-public abstract class IsInRangeConverter<TObject> : BaseConverterOneWay<IComparable, object>
+public abstract class IsInRangeConverter<TValue, TReturnObject> : BaseConverterOneWay<TValue, object> where TValue : IComparable
 {
-	/// <summary>
-	/// Bindable property for <see cref="MinValue"/>
-	/// </summary>
-	public static readonly BindableProperty MinValueProperty = BindableProperty.Create(nameof(MinValue), typeof(IComparable), typeof(IsInRangeConverter<TObject>));
-
-	/// <summary>
-	/// Bindable property for <see cref="MaxValue"/>
-	/// </summary>
-	public static readonly BindableProperty MaxValueProperty = BindableProperty.Create(nameof(MaxValue), typeof(IComparable), typeof(IsInRangeConverter<TObject>));
-
-	/// <summary>
-	/// Bindable property for <see cref="TrueObject"/>
-	/// </summary>
-	public static readonly BindableProperty TrueObjectProperty = BindableProperty.Create(nameof(TrueObject), typeof(TObject?), typeof(IsInRangeConverter<TObject>));
-
-	/// <summary>
-	/// Bindable property for <see cref="FalseObject"/>
-	/// </summary>
-	public static readonly BindableProperty FalseObjectProperty = BindableProperty.Create(nameof(FalseObject), typeof(TObject?), typeof(IsInRangeConverter<TObject>));
-
 	/// <inheritdoc/>
 	public override object DefaultConvertReturnValue { get; set; } = new();
 
-	/// <summary>Minimum value.</summary>
-	public IComparable? MinValue
-	{
-		get => (IComparable?)GetValue(MinValueProperty);
-		set => SetValue(MinValueProperty, value);
-	}
+	/// <summary>The object that corresponds to False value.</summary>
+	public TReturnObject? FalseObject { get; set; }
```

Comment:
This is a breaking change, are we sure we want to drop the ability to make the following properties bindable?

- `FalseObject`
- `MaxValue`
- `MinValue`
- `TrueObject`

Reply:
I was copying the changes that you made to `CompareConverter` to bring this converter in-to-line.  As such, I believe it will have the same impact.